edx_proctoring/locale/es_419/LC_MESSAGES/django.po

```diff
@@ -341,6 +343,8 @@ msgid ""
 "        You will be guided through steps to set up online proctoring software and to perform various checks.</br>\n"
 "      "
 msgstr ""
+"\n"
+"Será guiado en los pasos para establecer el programa de revisión online y como hacer varias verificaciones."
```

Comment:
Errors with these translations. Took a look in Transifex, will let it rerun next week.